group-service/src/main/java/hcmus/alumni/group/controller/GroupServiceController.java

```diff
@@ -912,4 +1121,110 @@ public ResponseEntity<String> deletePostGroupReaction(@RequestHeader("userId") S
 		postGroupRepository.reactionCountIncrement(id, -1);
 		return ResponseEntity.status(HttpStatus.OK).body(null);
 	}
+	
+	@GetMapping("/{id}/votes/{voteId}")
+	public ResponseEntity<HashMap<String, Object>> getVoteUsers(
+			@RequestHeader("userId") String userId,
+			@PathVariable(name = "id") String postId,
+			@PathVariable Integer voteId,
+			@RequestParam(value = "page", required = false, defaultValue = "0") int page,
+			@RequestParam(value = "pageSize", required = false, defaultValue = "50") int pageSize) {
+		if (pageSize == 0 || pageSize > MAXIMUM_PAGES) {
+			pageSize = MAXIMUM_PAGES;
+		}
+		HashMap<String, Object> result = new HashMap<String, Object>();
+
+		Pageable pageable = PageRequest.of(page, pageSize, Sort.by(Sort.Direction.fromString("desc"), "createAt"));
+		Page<IUserVotePostGroupDto> users = userVotePostGroupRepository.getUsers(voteId, postId, pageable);
+
+		result.put("totalPages", users.getTotalPages());
+		result.put("users", users.getContent());
+
+		return ResponseEntity.status(HttpStatus.OK).body(result);
+	}
+
+	@PreAuthorize("hasAnyAuthority('Group.Vote')")
+	@PostMapping("/{id}/votes/{voteId}")
```

Comment:
```suggestion
	@PostMapping("/{postId}/votes/{voteId}")
```
nên ghi rõ ra là id của gì để tránh gây hiểu lầm
id ko thì sẽ là của group
bên dưới cũng v